src/components/Menu/Menu.test.tsx

```diff
@@ -109,6 +109,24 @@ describe("Menu allows selection", () => {
     fireEvent.click(openButton);
     expect(button1).toHaveAttribute("data-testid", "selected-item");
   });
+
+  it("displays selected item as label", () => {
+    render(
+      <Menu
+        showSelectionAsLabel
+        labelText={"Menu"}
+        listItemsData={defaultListItems}
+      />
+    );
+    const openButton = screen.getByText("Menu");
+    fireEvent.click(openButton);
+    const button1 = screen.getByText("I'm item 1").closest("button");
+    fireEvent.click(button1);
+    fireEvent.mouseDown(document.body);
```

Comment:
Why do you need this?

Reply:
Not totally sure which line you mean here, but 125 is just one of the ways to close the menu, and lines 126-128 are just confirming that the item is selected and that the label of the item appears twice, first as the menu label and then as the item label. @EdwinGuzman

Reply:
It's for line 125. The menu closes after line 124 and just curious why the need to focus outside of the menu.